tests/testthat/_snaps/S3.md

```diff
@@ -3,7 +3,7 @@
     Code
       S3_class(c("ordered", "factor"))
     Output
-      S3 class <ordered/factor>
+      <R7_S3_class>: S3<ordered>
```

Comment:
It seems to use `class_desc()` now. So we don't show all classes anymore?

Reply:
What do you think? I don't have any strong feelings either way except that it seems better to be consistent? (i.e. any change should be to `class_desc()`)